### DISCUSSION TITLE
Планируется ли добавление управления через MQTT?

### DISCUSSION THREAD

**Discussion by tmkbl** [vvip-68/LedPanelWiFi#7] [Общие вопросы]
Весьма полезная вещь - управление через MQTT. Будет ли она добавлена в прошивку в будущем?

**Comment by vvip-68** [comment]
Нет достаточно квалифицированных потребителей, которые могли бы четко и внятно объяснить что они хотят от MQTT, какие топики в каком формате должны передаваться / получаться и так далее.

У меня нет необходимости (и возможности) управлять гирляндой на окне (или настольными часами) через MQTT. Нет внятной постановки задачи - нет и реализации.
